rewrite-java/src/main/java/org/openrewrite/java/AddCommentToMethod.java

```diff
@@ -81,22 +81,22 @@ public TreeVisitor<?, ExecutionContext> getVisitor() {
         return Preconditions.check(new DeclaresMethod<>(methodMatcher), new JavaIsoVisitor<ExecutionContext>() {
             @Override
             public J.MethodDeclaration visitMethodDeclaration(J.MethodDeclaration method, ExecutionContext ctx) {
-                J.MethodDeclaration md = super.visitMethodDeclaration(method, ctx);
+                J.MethodDeclaration m = super.visitMethodDeclaration(method, ctx);
```

Comment:
This change is just to make it easier to diff `AddCommentToMethod` and `AddCommentToMethodInvocation`.